package.json

```diff
@@ -37,8 +37,7 @@
   },
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "npm test"
```

Comment:
I think we should leave it. I know it's disruptive at the moment but after all Test are "green" again this is very usefull.